src/ui/wxWidgets/AddEditPropSheetDlg.cpp

```diff
@@ -2446,8 +2475,39 @@ Command* AddEditPropSheetDlg::NewEditEntryCommand()
   time_t t;
   time(&t);
   if (changes & Changes::Password) {
-    m_Item.UpdatePassword(tostringx(m_BasicPasswordTextCtrl->GetValue()));
-    m_Item.SetPMTime(t);
+    BaseEntryParms pl;
+    pl.InputType = m_Item.GetEntryType();
+    auto password = tostringx(m_BasicPasswordTextCtrl->GetValue());
+    if (m_Core.ParseAliasPassword(password, pl)) {
+      // Core validations:
```

Comment:
The validation code below is the same as in `NewAddEntryCommand` (and in `CAddEdit_Basic::OnApply`) - could we extract a common method (or at the very least have a method to construct the GTU string so it's consistent)?